DataConnectors/Zimperium MTD Connector.md

```diff
@@ -0,0 +1,41 @@
+# Connect your Zimperium Mobile Threat Defense to Azure Sentinel

```

Comment:
Remove the md file from this PR and please email me the md file - The documentation GitHub repo is different and not in this repo

Reply:
I have removed that file and sent via email as requested.